mlrun/projects/project.py

```diff
@@ -153,6 +155,10 @@ def new_project(
             raise ValueError("template must be a path to .yaml or .zip file")
         project.metadata.name = name
     else:
+        if override:
```

Comment:
Why is this on the `else` clause of checking `from_template`? Cannot the user create a new project from template and override existing?